src/app/pages/StoryPage/index.test.jsx

```diff
@@ -594,6 +600,28 @@ describe('Story Page', () => {
 
   it('should not render canonical ad bootstrap on amp', async () => {
     process.env.SIMORGH_APP_ENV = 'test';
+    const observers = new Map();
```

Comment:
Did the optimizely experiment code do something weird to break this?

Reply:
This test causes the optimizely event tracking tests to break. Started happening when the article complete stuff was added (https://github.com/bbc/simorgh/blob/61468c175affc2c41c1533dc956331a4dfa6282e/src/app/containers/OptimizelyArticleCompleteTracking/index.jsx#L29). 

I mocked the observer similar to how it is done in the tests for article complete tests (https://github.com/bbc/simorgh/blob/61468c175affc2c41c1533dc956331a4dfa6282e/src/app/containers/OptimizelyArticleCompleteTracking/index.test.jsx#L22).

Reply:
Having discussed it, there might be a simpler solution than mocking this.

Reply:
Can't be moved due to linting.